include/awkward/array/RawArray.h

```diff
@@ -384,6 +384,10 @@ namespace awkward {
       return std::vector<std::string>();
     }
 
+    int64_t list_depth() const override {
+      throw std::runtime_error("FIXME: RawArrayOf<T>::list_depth() is not implemented");
```

Comment:
Isn't the `list_depth` of a `RawArray` equal to `1`?

Reply:
yes, fixed. Thanks!